src/open_inwoner/accounts/tests/factories.py

```diff
@@ -34,6 +34,12 @@ class DigidUserFactory(UserFactory):
     is_prepopulated = True
 
 
+class eHerkenningUserFactory(UserFactory):
+    login_type = LoginTypeChoices.eherkenning
+    kvk = "12345678"  # TODO ideally this should be random (but still valid)
```

Comment:
I would remove the comment. The `DigidUserFactory` has the same comment, which is from December last year. Who is going to implement this and when?